src/components/atoms/ActionButton.js

```diff
@@ -5,16 +5,35 @@ import PropTypes from "prop-types";
  * Button component which is used to trigger actions on the page
  */
 export function ActionButton(props) {
```

Comment:
You should make sure to add the relevant states to the story for this component as well as the tests

Reply:
I have updated the stories and test, as well as the mobile view, please check it out